src/components/table/table.tsx

```diff
@@ -648,9 +648,9 @@ export class Table {
             >
                 <limel-checkbox
                     onChange={this.selectAllOnChange}
-                    checked={this.tableSelection.hasSelection}
+                    checked={this.tableSelection?.hasSelection}
                     indeterminate={
-                        this.tableSelection.hasSelection &&
+                        this.tableSelection?.hasSelection &&
                         this.selection.length < this.data.length
```

Comment:
Selection too, right? (data is always set to an empty array)

```suggestion
                        this.selection?.length < this.data.length
```

Reply:
fixed! 🙂